src/accsr/remote_storage.py

```diff
@@ -122,7 +136,7 @@ def _pull_object(
         remote_object: RemoteObjectProtocol,
         destination_path: str,
         overwrite_existing=False,
-    ) -> bool:
+    ) -> None:
         """
         Download the remote object to the destination path. Returns True if file was downloaded, else False
```

Comment:
you should either adjust the signature to return a bool as before (why was it changed?) or adjust the docstring

Reply:
Deleted last sentence